community/detectors/xwiki_cve_2024_21650/src/main/java/com/google/tsunami/plugins/detectors/rce/cve202421650/Cve202421650DetectorBootstrapModule.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.tsunami.plugins.detectors.rce.cve202421650;
+
+import com.google.inject.Provides;
+import com.google.tsunami.plugin.PluginBootstrapModule;
+import com.google.tsunami.plugins.detectors.rce.cve202421650.Annotations.OobSleepDuration;
+
+/** A {@link PluginBootstrapModule} for {@link Cve202421650Detector}. */
+public final class Cve202421650DetectorBootstrapModule extends PluginBootstrapModule {
+
+  @Override
+  protected void configurePlugin() {
+    registerPlugin(Cve202421650Detector.class);
+  }
+
+  @Provides
+  @OobSleepDuration
+  int provideOobSleepDuration(Cve202421650DetectorConfigs configs) {
+    if (configs.oobSleepDuration == 0) {
+      return 3;
```

Comment:
```suggestion
      return 10;
```

I think at least 10 seconds here is a safe bet, considering the network latency of a real environment and that the target may not respond quickly

Reply:
Updated